### PR TITLE
Use correction matrix for bone tail nodes

### PR DESCRIPTION
**Pull request by jjcasmar**

### CODE REVIEW

**Comment by jjcasmar** [comment]
I have tested with this two blender scenes. 
LBS.blend uses Linear Blend Skinning, so is not affected by this commit.
MechChain.blend uses direct parenting of the objects to the bone tails. This commit fixes this case when there is a rotation to convert Z to Y (checkable option in the exporter settings). 

[Scenes.tar.gz](https://github.com/KDAB/kuesa-blender/files/2830199/Scenes.tar.gz)

File: addons/kuesa_exporter/glTF-Blender-Exporter/scripts/addons/io_scene_gltf2/gltf2_generate.py [hunk 1]
```diff
@@ -1855,10 +1855,10 @@ def generate_nodes(operator,
 
                         # Generate the bone tail for direct parenting
                         distance_head_to_tail = (blender_bone.head - blender_bone.tail).length
+                        head_to_tail_matrix = mathutils.Matrix.Translation((0, distance_head_to_tail, 0))
                         tail_node = {}
+                        generate_node_parameter(export_settings, head_to_tail_matrix * axis_basis_change.transposed(),  tail_node, 'JOINT')
```
Comment (lemirep) [inline addons/kuesa_exporter/glTF-Blender-Exporter/scripts/addons/io_scene_gltf2/gltf2_generate.py]: Is there a point in keeping the other generate_node_parameter call a few lines above (line 1850)?

Comment (lemirep) [inline addons/kuesa_exporter/glTF-Blender-Exporter/scripts/addons/io_scene_gltf2/gltf2_generate.py]: If I understand correctly, this entirely bypasses matrix_basis and correction_matrix_local. Are you sure we don't need them?

Comment (jjcasmar) [reply addons/kuesa_exporter/glTF-Blender-Exporter/scripts/addons/io_scene_gltf2/gltf2_generate.py]: I must say Im not really sure about this. I have done several tests and it worked only when using the transpose of the axis_basis_change. I havent read the whole exporter to fully unedrstand how all the conversion from Y to Z is performed...

Comment (lemirep) [reply addons/kuesa_exporter/glTF-Blender-Exporter/scripts/addons/io_scene_gltf2/gltf2_generate.py]: Alright, I guess we'll see if that causes other problems

**Pull request merged**